modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/connection/ElasticsearchInstanceSettingsBuilder.java

```diff
@@ -305,7 +305,11 @@ private void _loadDefaultConfigurations() {
 
 		_configureTestMode();
 
-		put("transport.type", "netty4");
+		put("xpack.ml.enabled", false);
+		put("xpack.monitoring.enabled", false);
+		put("xpack.security.enabled", false);
+		put("xpack.sql.enabled", false);
+		put("xpack.watcher.enabled", false);
```

Comment:
Moving conversation here from Slack:

It appears that modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/connection/ElasticsearchInstanceSettingsBuilder.java is also used for the remote connection which can break the security integration and other stuff since now we are disabling a bunch of features of X-Pack. We should only disable those for the sidecar connection.

I maybe wrong though, better to double-check it.

Reply:
Thanks for double checking, @joshuacords :+1: